tools/fio-engine/xnvme_fioe.c

```diff
@@ -191,7 +191,7 @@ static struct fio_option options[] = {
 		.lname  = "xNVMe Backend",
 		.type   = FIO_OPT_STR_STORE,
 		.off1   = offsetof(struct xnvme_fioe_options, xnvme_be),
-		.help   = "Select backend. cmd-interface: []",
+		.help   = "Select backend.",
```

Comment:
great

Reply:
This is introduced in 229cb6fc8c2d99979515e0770f0cfb8b3500ed46
It should be squashed into it